packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1078,7 +1078,7 @@ def test_if_minuteman_routes_to_vip(cluster, timeout=125):
         'healthChecks': [{
             'protocol': 'COMMAND',
             'command': {
-                'value': 'test "$(curl -o /dev/null --max-time 5 -4 -w \'%{http_code}\' -s http://localhost:${PORT0}/|cut -f1 -d" ")" == 200'  # noqa
+                'value': 'curl -f -X GET http://$HOST:$PORT0/'
```

Comment:
This is a good question. It seemed like this test (test_if_minuteman_routes_to_vip) was failing consistently on azure (whereas it is just slightly flaky everywhere else) and I asked @sargun if he knew what was up. His response then was trying to change the health check to this. I got three fails in a row on this case before adding this check in, so it seems like it might be needed here...